packages/core/src/interfaces/axis-scales.ts

```diff
@@ -57,4 +57,5 @@ export interface AxesOptions {
  */
 export interface TimeScaleOptions {
 	addSpaceOnEdges?: boolean;
+	timeRangeToExtend?: number;
```

Comment:
This is confusing now, if `timeRangeToExtend` `"Only extends the axes if the timerange is greater than this amount"`, then what will `addSpaceOnEdges` do if the timerange is not greater than the one specified in `timeRangeToExtend`

Reply:
it will not add space on edges.  It will only do that if greater than the time range to extend